### PR TITLE
Resolver GROUP BY alised column

### PR DESCRIPTION
**Pull request by aljazerzen**

### CODE REVIEW

File: tests/test_sql_query.py [hunk 1]
```diff
@@ -504,6 +504,33 @@ async def test_sql_query_33(self):
         # this numbers change, so let's just check that there are 6 of them
         self.assertEqual(len(res[0]), 6)
 
+    async def test_sql_query_34(self):
+        # GROUP and ORDER BY aliased column
+
+        res = await self.squery_values(
+            """
+            SELECT substr(title, 2, 4) AS itl, count(*) FROM "Movie"
```
Comment (msullivan) [inline tests/test_sql_query.py:512]: :laughing:

File: tests/test_sql_query.py [hunk 2]
```diff
@@ -504,6 +504,33 @@ async def test_sql_query_33(self):
...
+            GROUP BY itl
+            ORDER BY itl
+            """
+        )
+        self.assertEqual(res, [["avin", 1], ["orre", 1]])
+
+    async def test_sql_query_35(self):
```
Comment (msullivan) [inline tests/test_sql_query.py:519]: Do these tests actually test an ambiguous case?

**Review by msullivan — APPROVED** [review]
So much SQL

File: tests/test_sql_query.py (hunk 2 above)
Comment (aljazerzen) [reply tests/test_sql_query.py:519]: No, it's not ambiguous, this all valid SQL. It's just that SQL has very particular rules on what names/aliases are allowed in which clauses. And that some tools rely on this behavior.

**Pull request merged**